src/js/components/sidebar.jsx

```diff
@@ -327,6 +384,21 @@ export default class SideBar extends React.Component {
     browser.tabs.remove(this._getSelectedTabIds());
   }
 
+  _reloadSelected() {
+    let selectedTabIds = this._getSelectedTabIds();
+
+    for (let id of selectedTabIds) {
```

Comment:
Use `const` over `let` here.

Each iteration through the loop creates a new binding, so you should always use `const ... of`, unless you need to mutate the value.m